domains/amrit.json

```
@@ -0,0 +1,9 @@
+{
+    "owner": {
+      "username": "amritsapkotadev",
+      "email": "amritsapkota.dev@gmail.com"
+    },
+    "records": {
+      "CNAME": "https://amritsapkota.netlify.app/"
```

remove the "https://" and the "/"
```suggestion
      "CNAME": "amritsapkota.netlify.app"
```